collect.go

```diff
@@ -223,6 +223,16 @@ func FeedWithErrs[T any](executor Executor, receiver func(context.Context, T) er
 	return making
 }
 
+// groupError returns the error associated with a group's context; if the error
+// was errGroupDone, that doesn't count as an error and nil is returned instead.
+func groupError(ctx context.Context) error {
+	err := context.Cause(ctx)
+	if err == errGroupDone {
```

Comment:
To cover wrapped errors as well I think it's preferable to use `errors.Is` or `errors.As` rather than `==` although I'm not entirely sure how wrapped errors might come into play here. (https://go.dev/wiki/ErrorValueFAQ)

Reply:
We explicitly don't want to find wrapped errors here, we want to see exactly this special error value which is the only value that means "no error"

Reply:
i can comment that nuance here if you'd prefer